File: backend/grafioschtrader-common/src/main/java/grafioschtrader/entities/Auditable_.java
package grafioschtrader.entities;

import java.util.Date;

import jakarta.annotation.Generated;
import jakarta.persistence.metamodel.SingularAttribute;
import jakarta.persistence.metamodel.StaticMetamodel;

@Generated(value = "org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor")
@StaticMetamodel(Auditable.class)
public abstract class Auditable_ {

  public static volatile SingularAttribute<Auditable, Date> lastModifiedTime;
  public static volatile SingularAttribute<Auditable, Date> creationTime;
  public static volatile SingularAttribute<Auditable, Integer> createdBy;
  public static volatile SingularAttribute<Auditable, Integer> lastModifiedBy;
  public static volatile SingularAttribute<Auditable, Integer> version;

}

File: frontend_16_failed/src/app/entities/auditable.ts
import {ProposeTransientTransfer} from './propose.transient.transfer';

export abstract class Auditable extends ProposeTransientTransfer {
  createdBy: number = null;
  creationTime: number = null;
  lastModifiedBy: number = null;
  lastModifiedTime: number = null;
  version: number = null;
}
